tests/Spec.hs

```diff
@@ -223,35 +233,36 @@ main = hspec $ do
       Project.integrationStatus pr `shouldBe` Project.NotIntegrated
 
     it "handles PullRequestClosed" $ do
-      let event1 = PullRequestOpened (PullRequestId 1) (Branch "p") (Sha "abc") "title" "peter"
-          event2 = PullRequestOpened (PullRequestId 2) (Branch "q") (Sha "def") "title" "jack"
+      let event1 = PullRequestOpened (PullRequestId 1) (Branch "p") (BaseBranch "master") (Sha "abc") "title" "peter"
+          event2 = PullRequestOpened (PullRequestId 2) (Branch "q") (BaseBranch "master") (Sha "def") "title" "jack"
```

Comment:
Just a general comment in this file, the lines are getting quite long, could you maybe introduce some newlines or move some things to variables (like a global `masterBranch = BaseBranch "master"` at the top of this file, that would safe a lot of redundant code).